engine/gamesys/src/gamesys/scripts/script_factory.cpp

```diff
@@ -239,14 +239,14 @@ namespace dmGameSystem
      * ```
      */
     static int FactoryComp_CreateWithMessage(lua_State* L, dmGameObject::HCollection collection, dmMessage::URL* receiver,
-                                            uint32_t index, dmhash_t id, dmGameObject::HPropertyContainer properties,
+                                            dmhash_t id, dmGameObject::HPropertyContainer properties,
                                             const dmVMath::Point3& position, const dmVMath::Quat& rotation, const dmVMath::Vector3& scale)
     {
         uint8_t DM_ALIGNED(16) buffer[512];
 
         dmGameSystemDDF::Create* create_msg = (dmGameSystemDDF::Create*)buffer;
         create_msg->m_Id = id;
-        create_msg->m_Index = index;
+        create_msg->m_Index = dmGameObject::INVALID_INSTANCE_POOL_INDEX;
```

Comment:
An instance index will be assigned in comp_factory.cpp instead

Reply:
If this is the actual only place it's being used now, I suggest we remove it from gamesys_ddf.proto